clients/go/pkg/audit/interceptor.go

```diff
@@ -278,6 +278,40 @@ func (i *Interceptor) StreamInterceptor(srv interface{}, ss grpc.ServerStream, i
 	}
 	logReq.Payload.AuthenticationInfo = &capi.AuthenticationInfo{PrincipalEmail: principal}
 
+	// If JWT Validator hasn't been added, we don't handle justification JWTs, and just log
+	// without the justification.
+	if i.jwtValidator != nil {
+		// JWT Validator added, look for justification info
+		md, ok := grpcmetadata.FromIncomingContext(ctx)
+		if !ok {
+			return i.handleReturnStream(ctx, ss, handler, status.Errorf(codes.FailedPrecondition, "unable to retreive grpc metadata."))
+		}
+		vals := md.Get("justification_token")
```

Comment:
`justification-token`?

Reply:
this still uses the underscores everywhere right now, i plan to update that everywhere in a future PR.

Reply:
Ack